lib/takeaway.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'menu'
+require_relative 'order'
+require 'rubygems'
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :menu, :order, :order_time
+
+  def initialize
+    @menu = Menu.new
+    @order = Order.new
+  end
+
+  def show_menu
+    menu.show
+  end
+
+  def place_order(item, quantity = 1)
+    return 'This item is not available' unless available?(item)
+    order.add(item, quantity)
+  end
+
+  def show_order
+    order.overview
+  end
+
+  def checkout_order(price = 0)
+    order.checkout(price)
+    @order_time = Time.now + 3600
+    send_sms(number)
+  end
+
+  def reset_order
+    order.reset
+  end
+
+  def available?(item)
+    menu.listed?(item)
+  end
+
+  private
+
+  TWILIO_NUMBER = '+44 1442 796246'
+
+  def send_sms(number)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `number`. If it's necessary, use `_` or `_number` as an argument name to indicate that it won't be used. You can also write as `send_sms(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)